### PR TITLE
Votesmart rx

### PR DESCRIPTION
**Pull request by paige-kohn**

### CODE REVIEW

File: src/main/java/kohn/rx/votesmart/VoteSmartController.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    private Disposable disposable;
+
+    @Inject
+    public VoteSmartController(VoteSmartService service, Provider<VoteSmartView> viewProvider) {
+        this.viewProvider = viewProvider;
+        this.service = service;
+    }
+
+    public void loadStates() {
+        disposable = Observable.interval(0, 30, TimeUnit.SECONDS)
```
Comment (andrewoid) [inline src/main/java/kohn/rx/votesmart/VoteSmartController.java]: Does this need to refresh every 30 seconds? Are you waiting for new States to added to the country?

Copying code without thinking is really problematic.

File: src/main/java/kohn/kohn-mco152-votesmart-project.iml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (andrewoid) [inline src/main/java/kohn/kohn-mco152-votesmart-project.iml]: these files shouldn't be checked in

File: src/main/java/kohn/rx/votesmart/VoteSmartController.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                .observeOn(Schedulers.single())
+                .subscribe(this::setStateList, throwable -> throwable.printStackTrace());
+    }
+
+    private void setStateList(StateList list) {
+        List<InternalList.State> states = list
+                .getList()
+                .getState()
+                .stream()
+                .filter((state -> state.getStateId().startsWith("N")))
```
Comment (andrewoid) [inline src/main/java/kohn/rx/votesmart/VoteSmartController.java]: Why are you filtering?

File: src/main/java/kohn/rx/votesmart/VoteSmartController.java (hunk 1 above)
Comment (paige-kohn) [reply src/main/java/kohn/rx/votesmart/VoteSmartController.java]: I believe I initially had it to refresh every 10 days, tried with filters and without, debugging didn't get me anywhere either, and this was just the last thing I tried.

File: src/main/java/kohn/rx/votesmart/VoteSmartController.java [hunk 4]
```diff
@@ -23,60 +23,57 @@ public VoteSmartController(VoteSmartService service, Provider<VoteSmartView> vie
...
-        List<Elections.Election> elections = list
-                .stream()
-                .distinct()
-                .collect(Collectors.toList());
-        System.out.println(elections.get(0));
 
-        viewProvider.get().setElections(elections);
 
+    public void getBills() {
+        disposable = Observable.interval(0, 10, TimeUnit.SECONDS)
```
Comment (andrewoid) [inline src/main/java/kohn/rx/votesmart/VoteSmartController.java]: 10 seconds is pretty short

**Comment by paige-kohn** [comment]
Despite all these changes I still haven't figured out why the feed is continuous, which results in the data not displaying at all.

**Comment by andrewoid** [comment]
please fix the conflicts

**Pull request merged**